canonical-web-and-design/hiring.canonical.com
# Add headers

# Summary

- Add `Cache-Control: private` header to make sure we are not caching
- Configure requests to make sure we log every thing!

# QA

- `dotrun`
- Open the network tab and make sure that every thing that is served has the right header (Cache-control: private)
- Look at the logs there should be: `talisker.requests` logs in the terminal
